src/Uno.UI/Controls/CommandBarHelper.iOS.cs

```diff
@@ -12,7 +12,7 @@ public static class CommandBarHelper
 	{
 		internal static void SetNavigationBar(CommandBar commandBar, UIKit.UINavigationBar? navigationBar)
 		{
-			commandBar.GetRenderer(() => new CommandBarRenderer(commandBar)).Native = navigationBar ?? throw new ArgumentNullException(nameof(navigationBar));
+			commandBar.GetRenderer(() => new CommandBarRenderer(commandBar)).Native = navigationBar;
```

Comment:
In which conditions is this null ?

Reply:
It is set to `null` a couple of times in the same file ex: (https://github.com/unoplatform/uno/blob/23404809f725e9d439e31e43de337364f3e7bfc5/src/Uno.UI/Controls/CommandBarHelper.iOS.cs#L104)

Reply:
I see it at least on lines 51, 100, 109